### DISCUSSION TITLE
Not able to access the GPU runtime in google colab free version

### DISCUSSION THREAD

**Discussion by PuranikPranav** [mrdbourke/pytorch-deep-learning#1090] [General]
The GPU option is greyed out in the free version of google colab. Is it not available or do we have to enable it somehow?

**Comment by LuluW8071** [comment]
@PuranikPranav  You need to connect to a T4 GPU instance in order to access the GPU
Go to runtime menu option > change runtime type > **select T4 GPU**
![image](https://github.com/user-attachments/assets/e984efaf-5384-40ae-9fe1-7147f07a1dbf)
